src/components/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { Dispatch, SetStateAction } from 'react';
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  activeTodos: Todo[];
+  completedTodos: Todo[];
+  filter: Filter;
+  setFilter: Dispatch<SetStateAction<Filter>>;
+  deleteAllCompletedHandler: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  activeTodos,
+  completedTodos,
+  filter,
+  setFilter,
+  deleteAllCompletedHandler,
+}) => {
+  return (
```

Comment:
it's better practive
```suggestion
export const Footer: React.FC<Props> = (props) => {
const {
  activeTodos,
  completedTodos,
  filter,
  setFilter,
  deleteAllCompletedHandler,
} = props
  return (
```